test/unit/config/test__base.py

```diff
@@ -275,39 +275,58 @@ def test_container_volume_mounting_with_Z(mock_isdir, mock_exists, mock_makedirs
 
 
 @pytest.mark.parametrize('container_runtime', ['docker', 'podman'])
-def test_containerization_settings(tmpdir, container_runtime):
-    with patch('ansible_runner.config._base.BaseConfig.containerized', new_callable=PropertyMock) as mock_containerized:
-        rc = BaseConfig(private_data_dir=tmpdir)
-        rc.ident = 'foo'
-        rc.cmdline_args = ['main.yaml', '-i', '/tmp/inventory']
-        rc.command = ['ansible-playbook'] + rc.cmdline_args
-        rc.process_isolation = True
-        rc.runner_mode = 'pexpect'
-        rc.process_isolation_executable = container_runtime
-        rc.container_image = 'my_container'
-        rc.container_volume_mounts = ['/host1:/container1', 'host2:/container2']
-        mock_containerized.return_value = True
-        rc.execution_mode = BaseExecutionMode.ANSIBLE_COMMANDS
-        rc._prepare_env()
-        rc._handle_command_wrap(rc.execution_mode, rc.cmdline_args)
+def test_containerization_settings(tmpdir, container_runtime, mocker):
```

Comment:
Nit: it is best to use `tmp_path` in place of the deprecated `tmpdir` because it has a pathlib interface (and is recommended generally).

Reply:
Done.